uaclient/cli.py

```diff
@@ -568,17 +537,24 @@ def action_enable(args, cfg, **kwargs):
     valid_services_names = entitlements.valid_services(allow_beta=args.beta)
     ret = True
 
-    for entitlement in entitlements_found:
+    for ent_name in entitlements_found:
```

Comment:
I think adding an unit test for the case were we try to enable a beta service already enabled would be good. I know it will be a little redundant considering we do have an integration test for it, but since the price we pay to run an unit test is really cheap, I don't see a problem having them both, since the unit test will allow us to detect problems earlier.

Let me know if you agree with that